app/src/main/res/values-sw360dp-v13/preference.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<!--Created By Me to fix the appearance issue caused by migrating to AndroidX-->
+<resources xmlns:tools="http://schemas.android.com/tools">
+    <bool name="config_materialPreferenceIconSpaceReserved" tools:ignore="MissingDefaultResource,PrivateResource">false</bool>
```

Comment:
Seems like this should be overridden in the theme itself. Does it help to change `preferenceTheme` in `styles.xml` with a new style that contains this attribute?

```xml
<item name="iconSpaceReserved">false</item>

```

Reply:
This may work, I think. But this attribute doesn't work for PreferenceCategory (The title of the preference).

PS. Nvm about that comment "Created by me..". I have put there everywhere in my local copy just incase if anything breakes, I know exactly where to look at. 😄